main.cpp

```diff
@@ -52,84 +54,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: pig
+//  action 1: the pig oinks
+pig.oink();
+//  action 2: the pig squeels
+pig.squeel();
+//  action 3: the pig creates a communist dictatorship
+pig.creatCommunistDictatorship();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:razor
+//  action 1: the razor shaves someones's armpits
+razor.shaveArmpits();
+//  action 2: the razor shaves someone's beard
+razor.shaveBeard();
+//  action 3: the razor shaves someone's legs
+razor.shaveLegs();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:fridge
+//  action 1:the fridge refridgerates
+fridge.refridgerate();
+//  action 2:the fridge light turns on when opened
+fridge.turnOnLight();
+//  action 3:the fridge displays an overheating warning
+fridge.overheatWarning();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:trafficLight
+//  action 1: traffic light displays green 
+trafficLight.displayGreen();
+//  action 2: traffic light displays red
+trafficLight.displayRed();
+//  action 3: traffic light displays amber
+trafficLight.showAmber();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:calculator
+//  action 1: calculator adds
+calculator.add();
+//  action 2:calculator subtracts
+calculator.subtract();
+//  action 3: calculator multiplies
+calculator.multiply();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:breakmaker
+//  action 1: breakmaker stirs mixture
+breadmaker.stirMixture();
+//  action 2: breadmaker heats up
+breadmaker.heatUp();
+//  action 3: breadmaker automatically turns off
+breadmaker.autoTurnOff();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:kettle
+//  action 1: kettle starts boils
+kettle.startBoiling();
+//  action 2: kettle stops boiling 
+kettle.stopBoiling();
+//  action 3: kettle displays LED when on
+kettle.displayLed();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:oven
+//  action 1: oven heats up
+oven.heatUp();
+//  action 2: oven light turns off
+oven.turnOffIndicator();
+//  action 3: oven alarm sounds when time is up
+oven.soundTimerAlarm();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.